intelmq-manager/php/config.php

```diff
@@ -39,13 +39,10 @@
     fclose($pipes[2]);
     $paths_status = proc_close($proc);
     if ($paths_status == 0) {
-        $paths_output = json_decode($paths_stdout);
-        $FILES['bots'] = $output['BOTS_FILE'];
-        $FILES['defaults'] = $output['DEFAULTS_CONF_FILE'];
-        $FILES['harmonization'] = $output['HARMONIZATION_CONF_FILE'];
-        $FILES['pipeline'] = $output['PIPELINE_CONF_FILE'];
-        $FILES['runtime'] = $output['RUNTIME_CONF_FILE'];
-        $FILES['system'] = $output['SYSTEM_CONF_FILE'];
-        $FILES['positions'] = $output['CONFIG_DIR'] . "/manager/positions.conf";
-    }
+	    $output = json_decode($paths_stdout);
+	    foreach($output->paths as $path){
+		    $FILES[$path[0]]=$path[1];
+	    }    
```

Comment:
Why? This sets variables which are never used. The result is an array like this:
```php
Array
(
    [bots] => /opt/intelmq/etc/BOTS
    [defaults] => /opt/intelmq/etc/defaults.conf
    [harmonization] => /opt/intelmq/etc/harmonization.conf
    [pipeline] => /opt/intelmq/etc/pipeline.conf
    [runtime] => /opt/intelmq/etc/runtime.conf
    [system] => /opt/intelmq/etc/system.conf
    [positions] => /opt/intelmq/etc/manager/positions.conf
    [BOTS_FILE] => /opt/intelmq1/etc/BOTS
    [DEFAULTS_CONF_FILE] => /opt/intelmq1/etc/defaults.conf
    [HARMONIZATION_CONF_FILE] => /opt/intelmq1/etc/harmonization.conf
    [PIPELINE_CONF_FILE] => /opt/intelmq1/etc/pipeline.conf
    [RUNTIME_CONF_FILE] => /opt/intelmq1/etc/runtime.conf
    [VAR_RUN_PATH] => /opt/intelmq1/var/run/
    [STATE_FILE_PATH] => /opt/intelmq1/var/lib/state.json
    [DEFAULT_LOGGING_PATH] => /opt/intelmq1/var/log/
    [__file__] => /home/sebastianw/dev/intelmq/intelmq/bin/intelmqctl.py
    [CONFIG_DIR] => /opt/intelmq1/etc/
    [ROOT_DIR] => /opt/intelmq1/
)
```

Reply:
I'm working on a patch for intelmqctl to provide the data as dictionary, not as list

Reply:
Yes, there are a couple of variable it never uses, but some of them are used in the others scripts, like php/load_configs.php.
Im sure we could do it better but i dont know all the app and as i noticed you are changing to python i just wrote a simple fix!

Reply:
What I meant is, that your your wrote the string `/opt/intelmq1/etc/BOTS` (as an example for an alternative root) to the key `BOTS_FILE`, which is never used, but should be `bots` instead. Same for all other variables.